app/views/posts/index.html.erb

```diff
@@ -1 +1,22 @@
-<h1>Users Post</h1>
+<div class="container">
+  <%= render "shared/user_header", user: @user%>
+  <% @posts.each do |post| %>
+    <div class="w-50 border border-2 p-2">
+      <h4> <%= link_to post.title, user_post_path(post, user_id: @user.id) %> </h4>
+      <p> <%= post.text %> </p>
+      <div class="d-flex justify-content-end">
+        <span class="">Comments: <%= post.comments.count%> </span>, 
+        <span class="ml-2" > Comments: <%= post.likes.count%> </span>
+      </div>
+      <div class="border mt-1 p-2">
+        <% post.comments.each do |comment| %>
```

Comment:
- [ ] The comments to be rendered on this post index page should use the recent comments method you have in the Post model, kindly do that.